src/index.js

```diff
@@ -41,6 +41,9 @@ export function listen({
   // RethinkDB authKey for authenticated connections
   dbAuthKey = null,
 
+  // CA Certificate for the SSL connection, if required
```

Comment:
This comment is a bit inaccurate now that we're passing in the whole object instead of just cacert. Can you change it to say something like "For TLS connections to RethinkDB, equivalent to ssl argument in r.connect"?